exchange_cycle_testing.py

```diff
@@ -88,7 +88,7 @@ def standardize_path_start(path: List[str]) -> List[str]:
 def main():
     """Run several tests and dump the stats"""
 
-    ITERATIONS = 100000
+    ITERATIONS = 1000
```

Comment:
FYI, there are _at least_ 87 million possible valid paths that can be made from our current sample groups.